### PR TITLE
Visualize: fix flatten of observable mapping with one observable

### PR DESCRIPTION
**Pull request by Doresic**
Observable mapping visualization failed in the case of only one observable. Fixed to account for the case of one observable.

### CODE REVIEW

**Comment by stephanmg** [comment]
Looks good to me.

**Review by vwiela — APPROVED** [review]
Looks good, thanks for the fix.

**Review by PaulJonasJost — APPROVED** [review]
Should add a test for future cases probably

**Comment by dweindl** [comment]
I think there are two more such cases in `pypesto/visualize/observable_mapping.py` and one in `pypesto/visualize/ordinal_categories.py`.

File: test/visualize/test_visualize.py [hunk 1]
```diff
@@ -1240,3 +1241,54 @@ def test_parameters_correlation_matrix(result_creation):
...
+        / "doc"
+        / "example"
+        / "example_ordinal"
+        / "example_ordinal.yaml"
+    )
+    petab_problem = petab.Problem.from_yaml(example_ordinal_yaml)
+    # Set seed for reproducibility.
+    np.random.seed(0)
+    optimizer = pypesto.optimize.ScipyOptimizer(
+        method="L-BFGS-B", options={"maxiter": 10}
```
Comment (dweindl) [inline test/visualize/test_visualize.py:1261]: For just testing that visualization does not produce an error, a single iteration would suffice, right?

**Review by dweindl — APPROVED** [review]

**Review by stephanmg — APPROVED** [review]
Thanks for the contribution. From my side this can be merged.

**Pull request merged**